concent_api/core/models.py

```diff
@@ -518,3 +519,67 @@ def clean(self):
             raise ValidationError({
                 'pending_response': 'PaymentInfo should be related with Pending Response'
             })
+
+
+class GlobalTransactionState(Model):
+    """
+    Represents state of whole transaction service by storing last `nonce` used.
+
+    There should always be at most one object of this type with id = 0. If it does not exist, it should be created and
+    nonce initialized with a value obtained from SCI (Client.get_transaction_count()).
+    """
+
+    nonce = DecimalField(max_digits=32, decimal_places=0, default=0, unique=True)
+
+    def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
```

Comment:
Why does this warning need to be disabled here?

Reply:
PILINT gives warning if there is no disable:
`Parameters differ from overridden 'save' method (arguments-differ)`

Reply:
So please define it with the same parameters.